docs/ilm.asciidoc

```diff
@@ -0,0 +1,126 @@
+
+[options="header"]
+|=======================================================================
+|Event type |Hot |Warm
+|Errors & Spans
+|Rollover: `max_size: 50gb`, `max_age: 1 day`,
+`priority: 100`
+|`min_age: 7 days`,
+readonly,
+`priority: 50`
+|Transactions & Metrics
+|Rollover: `max_size: 50gb`, `max_age: 7 days`,
+`priority: 100`
+|`min_age: 31 days`,
+readonly,
+`priority: 50`
+|=======================================================================
+
+IMPORTANT: Changes to the default index lifecycle policy do not take effect until the current index has rolled over.
+If you'd like to manage a custom policy, see <<manual-ilm-setup,Manual ILM>>.
+
+[float]
+[[ilm-setup]]
+==== ILM default policy setup
+
+To set up index lifecycle management, set `ilm.enabled` to `true` in apm-server.yml.
+ILM can only be enabled for `output.elasticsearch`.
+When enabled, configurations defined for `output.elasticsearch.index` and `output.elasticsearch.indices` will be ignored.
+
+It is recommended to set up index lifecycle management (ILM) before starting APM Server.
+This excludes setup from the ingestion process, which allows you to ensure ILM is set up correctly before using APM.
+
+Run the <<setup-command>> with the ` --index-management` flag to set up the default ILM policy:
+
+[source,js]
+-----------------------
+./apm-server setup --index-management
+-----------------------
+// CONSOLE
+
+You can confirm the policy was created with the GET lifecycle policy API.
+Here's what the transaction response looks like:
+
+[source,js]
+-----------------------
+GET _ilm/policy
+{
+    "apm-7.2.0-transaction": {
+        "version": 1,
+        "modified_date": "2019-05-28T15:55:26.791Z",
+        "policy": {
+            "phases": {
+                "warm": {
+                    "min_age": "31d",<1>
+                    "actions": {
+                        "readonly": {},
+                        "set_priority": {
+                            "priority": 50
+                        }
+                    }
+                },
+                "hot": {
+                    "min_age": "0ms",
+                    "actions": {
+                        "rollover": {
+                            "max_size": "50gb",<2>
+                            "max_age": "7d"<3>
+                        },
+                        "set_priority": {
+                            "priority": 100<4>
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+-----------------------
+// CONSOLE
+<1> Move to warm phase after _31 days_
+<2> Rollover after _50gb_
+<3> Rollover after _7 days_
+<4> Priority for recovering your indices after a node restart. Higher priorities are recovered first.
+
+Your indices are now configured to use index lifecylce management. Go ahead and <<setting-up-and-running,run APM Server>>.
+
+// [float]
+// [[ilm-upgrade]]
+// ==== ILM default policy upgrades
```

Comment:
Having a paragraph declaring default policies are subject to change in future releases is a good idea, please add it.

Reply:
👍